doc/nrf/ug_multi_image.rst

```diff
@@ -134,35 +145,12 @@ The three options are:
 
 You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image (as specified in ``add_child_image``).
 
-The following example shows the configuration options for MCUboot:
+This can be done by including the :file:`Kconfig.template.build_strategy` template, as shown below.
 
 .. code-block:: Kconfig
 
-   choice
-  	prompt "MCUboot build strategy"
-  	default MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
-
-   config MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE
-  	# Mandatory option when being built through 'add_child_image'
-  	bool "Use HEX file instead of building MCUboot"
-
-   if MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE
-
-   config MCUBOOT_HEX_FILE
-  	# Mandatory option when being built through 'add_child_image'
-  	string "MCUboot HEX file"
-
-   endif # MCUBOOT_USE_HEX_FILE
-
-   config MCUBOOT_BUILD_STRATEGY_SKIP_BUILD
-  	# Mandatory option when being built through 'add_child_image'
-  	bool "Skip building MCUboot"
-
-   config MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
-  	# Mandatory option when being built through 'add_child_image'
-  	bool "Build from source"
-
-   endchoice
+   module=MCUBOOT
+   source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.build_strategy"
```

Comment:
If waiting for ongoing upmerge, then you'll get this in:
https://github.com/tejlmand/fw-nrfconnect-zephyr-1/commit/a861fc96b2bed0260d337599c32644d3075e1e10

meaning this can change to:
```suggestion
source "$(ZEPHYR_NRF_MODULE_DIR)/subsys/partition_manager/Kconfig.template.build_strategy"
```

Reply:
Yep, I will do that.